### PR TITLE
[RVV]: vectorized apply_l and apply_inv_l partially

### PR DESCRIPTION
**Pull request by kolobabka**

### CODE REVIEW

**Comment by Exactlywb** [comment]
Boost results in README?

File: tools/vectorized/vectorized.c [hunk 1]
```diff
@@ -21,16 +21,21 @@ void apply_x(Block *block, const Block *key) {
 
 // TODO: try to get a profit here
```
Comment (Exactlywb) [inline tools/vectorized/vectorized.c:22]: Remove TODO

File: tools/vectorized/vectorized.c [hunk 2]
```diff
@@ -21,16 +21,21 @@ void apply_x(Block *block, const Block *key) {
...
 void apply_l(Block *block) {
```
Comment (Exactlywb) [inline tools/vectorized/vectorized.c:23]: SIMD instructions are not readable at all. Maybe here is the reason to write down pseudo-code of what we're doing as a comment before function. Smth like this:

```
/*
  v1 = *block;
  v2 = v1 + {1};
  ...
*/
void apply_l(Block *block){...}
```

File: tools/vectorized/vectorized.c (hunk 1 above)
Comment (kolobabka) [reply tools/vectorized/vectorized.c:22]: Done

File: tools/vectorized/vectorized.c (hunk 2 above)
Comment (kolobabka) [reply tools/vectorized/vectorized.c:23]: Done

**Pull request merged**